apps/stress-test/scripts/utils/configureYargs.js

```diff
@@ -0,0 +1,119 @@
+const { getBrowsers } = require('./getBrowsers');
+
+/**
+ * @typedef {Object.<string, import('yargs').Options} YargsOptions
```

Comment:
Do this really work? O_o

Reply:
It passed type-check but I see a missing `>` so I'm not sure how thoroughly the JSDocs are actually being checked. 🤔🤔🤔

I have a follow up PR where I need to do some more work with types so I'll investigate further there.